pyuvdata/uvdata/tests/test_miriad.py

```diff
@@ -1225,14 +1162,14 @@ def test_read_write_read_miriad_partial_error_special_cases():
     )
 
     # cleanup
-    del uv_in, full
-    gc.collect()
-    shutil.rmtree(write_file)
+    # del uv_in, full
```

Comment:
As an aside, when we go through and get rid of these, we should make sure that we're closing any `_miriad.UV` objects with the new `.close()` method, rather than this hokey `del/gc.collect()` scheme.

Reply:
So I wrote `_miriad.UV.__dealloc__` to call `self.close()` and I think when you call `del miriad_object` it will automatically call the dealloc block. But I'm not sure how that works if it is a UVData object. :thinking: